airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -150,6 +161,22 @@ public static Source sshWrappedSource() {
     this.stateEmissionFrequency = INTERMEDIATE_STATE_EMISSION_FREQUENCY;
   }
 
+  private ConnectorSpecification modifySpec(final ConnectorSpecification originalSpec) {
+    final ConnectorSpecification spec = Jsons.clone(originalSpec);
+    final ObjectNode properties = (ObjectNode) spec.getConnectionSpecification().get("properties");
```

Comment:
Dumb question but is there any chance that the `properties` field is unset in the json and this ObjectNode is null?